packages/scene-composer/package.json

```diff
@@ -46,7 +46,7 @@
     "convert-svg": "npx @svgr/cli --out-dir src/assets/auto-gen/icons/ --typescript --index-template tools/index-template.js -- src/assets/icons/",
     "release": "run-s compile copy-assets",
     "copy-assets": "copyfiles -e \"**/*.tsx\" -e \"**/*.ts\" -e \"**/*.snap\" -e \"**/*.js\" -e \"**/*.jsx\" -e \"**/*.json\" \"src/**/*\" dist/",
-    "lint": "eslint . --max-warnings=575",
+    "lint": "eslint . --max-warnings=567",
```

Comment:
FYI to reviewers: After reviewing, this appear to have been increased due to the update in the interfaces.ts file where `KnownSceneProperty` was added to the `properties` field of  `ISceneDocument` & has been implemented in more places than it was previously. `any` has not increased in implementations, but did increase in occurrences from existing code that has been reused. This is a valid override to the ratchet.